podium-gateway/src/main/webapp/app/shared/request/request-review-feedback.ts

```diff
@@ -12,8 +12,12 @@ import { RequestReviewDecision } from './request-review-decision';
 
 export class RequestReviewFeedback {
 
-    decision?: RequestReviewDecision;
-    feedback?: string;
+    id?: string;
+    advice?: RequestReviewDecision;
+    reviewer?: string;
+    date?: Date;
+    summary?: string;
+    description?: string;
```

Comment:
Summary and description are the contents of the message object in a request review.